src/Sylius/Bundle/ResourceBundle/Controller/ResourceUpdater.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\Controller;
```

Comment:
I thought about that, but for me it doesn't make sense when only one from ten(?) collaborators be in separate directory. However, 👍  for refactoring `ResourceController` and its collaborators - but in different PR.